docs/pages/zero-trust-access/infrastructure-as-code/using-tctl.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: "Getting Started with tctl"
+description: Provides a conceptual overview of tctl, the Teleport administrative client tool.
+tags:
+- conceptual
+- platform-wide
+---
+
+This guide provides an overview of `tctl`, a command-line tool for managing
+Teleport dynamic resources.
+
+For an overview of what a dynamic resource is, see [Infrastructure as
+Code](infrastructure-as-code.mdx).
+
+For a comprehensive list of `tctl` commands, arguments, flag, and environment
+variables, see the [`tctl` CLI Reference](../../reference/cli/tctl.mdx).
+
+## When to use tctl
+
+`tctl` is a Teleport Auth Service client that manages dynamic resources. The
+following Teleport Auth Service clients can also manage dynamic resources:
+
+- [Teleport Terraform provider](./terraform-provider/terraform-provider.mdx)
+- [Teleport Kubernetes operator](./teleport-operator/teleport-operator.mdx)
+- [Custom API clients](../api/api.mdx)
+
+We recommend that you maintain your organization's dynamic Teleport resources as
+configurations in a code repository and apply them with the Terraform provider
+or Kubernetes operator.
+
+`tctl` is best suited for ad hoc operations such as retrieving data about your
+cluster or unanticipated configuration changes.
```

Comment:
I have mixed feeling about this recommendation. I think for most small Teleport customers, `tctl create -f` the files in a git repo is the best solution. Terraform and the operator make everything more complex. Large-scale users already know they should use IaC (and it's very likely required for compliance anyway).

```suggestion
`tctl` is best suited for ad hoc operations such as retrieving data about your
cluster or unanticipated configuration changes, and managing resources on
small scale Teleport clusters.

As the amount of Teleport resources increases, we recommend you to switch to
IaC tools such as Terraform or the Kubernetes operator as they offer more robust
resource lifecycle management.
```

Reply:
Thanks! Added in 3ba948ca04f1547ddcafc70abd1b49d1f18983ff